vespabase/src/common-env.sh

```diff
@@ -129,6 +129,35 @@ optionally_reduce_base_frequency() {
     fi
 }
 
+get_hugepage_size_mb() {
+    while read -r name size rest
+    do
+        if [[ "$name" =~ ^Hugepagesize:$ ]]
+        then
+          hugepagesize="$size"
+          unit="${rest,,}"
+          break
+        fi
+    done < /proc/meminfo
```

Comment:
I think we should only enable -XX:+UseTransparentHugePages if:

1. transparent huge pages is set to madvise (or defer+madvise?)

and then get the transparent huge page size from `/sys/kernel/mm/transparent_hugepage/hpage_pmd_size`.  `Hugepagesize:` from `/proc/meminfo` is actually a HugeTLB setting.

`hpage_pmd_size` depends on the CPU architecture.  On x64 it is hard coded to 2 MB.  On arm64/aarch64 it is a function of the kernel config CONFIG_ARM64_PAGE_SHIFT (subtracting 1 => 0.25x factor).